seong-geun/app.js

```diff
@@ -115,22 +129,55 @@ app.get("/u_posts", async (req, res) => {
    `,
     [userId]
   );
+
   res.status(200).json({ data: userspost });
-  const { id, profile_image, username, age, email } = req.body;
+});
 
+// 게시글 수정하기
+app.patch("/m_posts", async (req, res) => {
+  const { postingUserId, postingContent } = req.body;
   await appDataSource.query(
-    `INSERT INTO users(
-      id,
-      profile_image,
-      username,
-      age,
-      email
-		) VALUES (?, ?, ?, ?, ?);
-	`,
-    [id, profile_image, username, age, email]
+    `UPDATE
+        posts
+    SET
+        posting_content = ?
+    WHERE
+        posts.user_id = ? 
+    `,
+    [postingContent, postingUserId]
   );
 
-  res.status(201).json({ message: "userCreated" });
+  const m_userspost = await appDataSource.query(
+    `SELECT
+        users.id as userId,
+        users.username as userName,
+        posts.id as postingId,
+        posts.title as postingTitle,
+        posts.posting_content as postingContent
+    
+    FROM 
+        users
+    LEFT JOIN posts
+    ON users.id = posts.user_id
+    `
+  );
+
+  res.status(200).json({ data: m_userspost });
+});
+
+// 게시글 삭제하기
+app.delete("/posts", async (req, res) => {
+  const { postingId } = req.body;
```

Comment:
```suggestion
app.delete("/posts/:postingId", async (req, res) => {
  const { postingId } = req.params;
```
delete 메소드에는 body로 데이터를 보내지 않도록 해주세요. 이는 실제로 delete method를 사용할 때에는 body로 들어오는 데이터를 파싱하지 않도록 설계된 프레임워크나, 버전이 있기 때문에 꼭 사용해야한다면 유의하여 사용하되, 원칙적으로는 지양하는 것이 좋습니다. 의도와 다르게 동작할 수 있습니다.